tests/tracer/test_processor.py

```diff
@@ -0,0 +1,93 @@
+import mock
+
+from ddtrace import Span
+from ddtrace.filters import TraceFilter
+from ddtrace.internal.processor import TraceProcessor
+
+
+def test_no_filters():
+    tp = TraceProcessor([])
+    trace = [Span(None, "span1"), Span(None, "span2")]
+    spans = tp.process(trace)
+    assert spans == trace
```

Comment:
Would it make sense to check that a `TraceProcessor` with no filters yields the same input trace, rather than potentially a copy of it with the same content?

~~~ python
assert spans is trace
~~~

Reply:
I don't think it's part of the assumption/protocol of this class, so I'd say the current test is correct.

Reply:
Yeah I don't think we have to make this assumption